phpmailer/phpmailer/CVE-2016-10033.yaml

```diff
@@ -0,0 +1,8 @@
+title:     Remote Code Execution
+link:      https://github.com/PHPMailer/PHPMailer/releases/tag/v5.2.18
+cve:       CVE-2016-10033
+branches:
+    5.x:
+        time:     2016-12-23 23:40:00
+        versions: ['>=5.0.0', '<5.2.18']
```

Comment:
Okay, was not aware that only versions >=5 are available via composer. Fine then.